release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -3216,4 +3216,28 @@ $ oc adm release info 4.9.48 --pullspecs
 [id="ocp-4-9-48-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-49"]
+=== RHBA-2022:6678 - {product-title} 4.9.49 bug fix update
+
+Issued: 2022-09-29
+
+{product-title} release 4.9.49 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6678[RHBA-2022:6678] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6677[RHBA-2022:6677] advisory.
```

Comment:
Both redhat.com links bring me to a 404 page. Is that happening for you as well or just me?

Reply:
Thanks so much for your review! The links are showing a 404 page right now because the advisories have not yet `shipped live`. Once they are live the link should show. Thanks again!

Reply:
Aha! That makes sense, thank you for checking!